dev-docs/lit-release-process.md

```diff
@@ -0,0 +1,148 @@
+</blockquote>
+
+### Tl;dr
+
+1. Approve "Release / Release" GitHub action on the latest commit to release.
+1. Code review the automatically created "Version Packages" PR.
+1. Squash and merge the "Version Packages" PR.
+1. Approve the "Release / Release" workflow on the commit created by "Version Packages" PR. This publishes the new releases to NPM and GitHub.
+
+## Process
+
+1. Find the [commit you want to release](https://github.com/lit/lit/commits/main), and click on the yellow dot (or red cross) that represents GitHub Action checks. There should be a pending Github Action called "Release / Release (push)". Click "Details".
+
+![GitHub Actions UI showing the pending Release action. A link has the text "Review pending deployments"](./images/lit-release-process/release-github-action-review-pending.png)
+
+2. Click on the link "Review pending deployments" which will navigate you to a GitHub Action page for the Release workflow. Then click "Review deployments" and approve the pending deployment.
+
+![GitHub actions UI for the Release showing a "Review deployments" button](./images/lit-release-process/release-github-action-review-pending.png)
+
+![GitHub actions UI modal for approving a pending release](./images/lit-release-process/review-pending-release-modal-ui.png)
+
+3. The GitHub release action will now automatically generate a "Version Packages" PR.
+
+<blockquote>
+If a Version Packages pull request has not been created, an error may have occurred in the GitHub action run. Look at the full logs for root cause.
+
+Possible failures previously encountered: A malformed changest file referencing a package name that doesn't exist.
+The fix was to commit the fixed changeset file to main, and re-run the release process on that new commit with the fixed changeset.
+
+</blockquote>
+
+![A screenshot of a GitHub pull request title "Version Packages"](./images/lit-release-process/version-packages-pr.png)
+
+4. Request code reviews from members of the team familiar with the packages that will be released.
+
+   1. The purpose of this pull request is to:
+      1. Delete all the changeset files, moving their contents into the various Changelog files.
+      1. Bump the `window.litElementVersions` global.
+      1. Bump version numbers in package.json files.
+      1. Bump dependency ranges in related package.json files if needed. (Only expected in minor or major releases).
+   1. Errors to look for:
+      1. When a new package is created and released, it is easy to accidentally have the version in the package.json set to vesion 1.0.0, resulting in Changeset bumping the version to 2.0.0. Fix is to create the package at version 0.0.0.
+      1. Major version releases or minor features may require further scrutiny as major releases mark major events and require coordination.
+      1. Changelog updates to `lit` core libraries such as `lit-html` should be relected in the umbrella `lit` package.
```

Comment:
We should be specific here and say `lit-html`, `lit-element`, and `@lit/reactive-element` are considered "core".

Reply:
Done!